qc-runner/src/data-ingest/0-download_gem2s.r

```diff
@@ -0,0 +1,54 @@
+require("RJSONIO")
+require("paws")
+require("zeallot")
+require("ids")
+
+task <- function(input,pipeline_config) {
+    # you are receiving the full sample object instead of sample names
+    project_id <- input$projectId
+    sample_names <- input$sampleNames # extract sample names from samples object
```

Comment:
can we just use the sample's uuid as the sample name? Sample UUIDs may not be changed, while names are easily changed via data-management page. By using sample uuid, we allow users to change sample names without having to re run data-ingest.